### PR TITLE
Decimal point

### PR DESCRIPTION
**Pull request by tirix**

### CODE REVIEW

File: set.mm [hunk 1]
```diff
@@ -489988,88 +490285,231 @@ of the indicator function (Contributed by Thierry Arnoux,
...
+    UYRGFLMUYBXCYRMLFYRXCUYCUYDUYEYDUYFYDDVUOUDHZUYRCGHZOHZXVEUYROHXUNWYOXVFUYR
+    XVEOUYRWUNUYGYDXVGDVUPUDHXVAOHDVUQUDHXUOOHXUNDVUOCEVBMVUNVSWYPWEZWOCVAUYHEW
+    HUYHZYTDVUPELVBDVUOVBXVHWEZUXSXVIXVDYTDVUQLMVBDVUPVBXVJWEUYIXVDWXKYTUYJMVUN
+    VSWYPUYMUYKUYJUYKUYLVVEWYKPBWYAPBZDWYASJZKWYLWYNTUJWURWVAVUSUYRVWNUQXUHVPZW
+    VTXJXVKXVLWYIWYAWYHYIULMWYKWYAUYNVKXMWYBPBWURPBWWOWYEWYLTWYJXVMWWQWYBWURWVA
+    UYOVKXMWVBWYCWURWXTWYBWVAWYJWVTWXSVPXVMXQRVUKWVBWURVWJWUTWVAWXRWVSWWBWWQYOW
+    XSVPXVMXQRVWCVWLVVLVYMKZWUGWUSTVWJVWNVYLXVNVYOVYNWFVUKVUSUYRUYNVKXMVULWUEVU
+    SVUEVUKVVGVWJXJUYRVUKUQVWJXJVWNXQRUOUYP $.
+
+  $( Decimal multiplicatin galore! (Contributed by Thierry Arnoux,
```
Comment (benjub) [inline set.mm:490371]: multiplication

**Review by benjub — APPROVED** [review]

**Comment by benjub** [comment]
Using ax-1cn instead of ax1cn should do the trick.

File: set.mm [hunk 2]
```diff
@@ -466016,6 +466060,70 @@ its graph has a given second element (that is, function value).
     dpval3 $p |- ( A . B ) = _ A B $=
       ( cdp co c1 cc0 cdc cdiv caddc cdp2 dpval2 df-dp2 eqtr4i ) ABEFABGHIJFKFA
       BLABCDMABNO $.
+
+    $( Multiply by ` ; 1 0 ` a decimal expansion.  (Contributed by Thierry
+       Arnoux, 25-Dec-2021.) $)
```
Comment (avekens) [inline set.mm:466065]: I think we should use 10 instead of ` ; 1 0 `  within a comment.

**Review by avekens — APPROVED** [review]

**Review by avekens — CHANGES_REQUESTED** [review]
CI checks  failed: "hgt750lem2" should not use ax1cn, but ax-1cn instead.
OK, @benjub said this already before.

File: set.mm (hunk 2 above)
Comment (avekens) [reply set.mm:466065]: Ok now

**Review by avekens — APPROVED** [review]

**Review by benjub — APPROVED** [review]

**Comment by david-a-wheeler** [comment]
@tirix - thanks! I'm hoping that eventually decimal points will move into the main body, since we now have multiple users. I hear other people sometimes use decimal points :-).

**Pull request merged**